lib/msal-core/karma.conf.js

```diff
@@ -4,32 +4,33 @@ const path = require('path');
 module.exports = function (config) {
     config.set({
         basePath: '',
-        frameworks: ['jasmine-ajax', 'jasmine'],
+        frameworks: ['mocha', 'chai'],
         files: [
             'node_modules/babel-polyfill/dist/polyfill.js',
-            'tests/**/*.ts',
+            'tests/mochaTests/**/*.ts',
         ],
         preprocessors: {
-            'tests/**/*.ts': ['webpack', 'sourcemap'],
+            'tests/mochaTests/**/*.ts': ['webpack', 'sourcemap'],
```

Comment:
why mocha tests folder? cant we just have a /test folder?

Reply:
Temporary until all tests have been migrated. This is resolved in a future PR

Reply:
Didn't want to make this PR longer than it needed to be :)